apps/table.html

```diff
@@ -657,6 +687,38 @@ <h3 class="text-center h3 mb-0">Available Slides</h3>
 		}
 	}
 
+	function isValidHttpUrl(urlstring) {
+		try {
+		  url = new URL(urlstring);
+		} catch (_) {
+		  return false;  
+		} 
+		return url.protocol === "http:";
+	  }
+
+	function urlInputChange()
+	{
+		const urlInput = $('#urlInput');
+		const url = urlInput.val();
+
+		if (!isValidHttpUrl(url)) {
+			urlInput.addClass('is-invalid');
+			urlInput.parent().append(`<div class="invalid-feedback" id="inputUrl-feedback0">
+					Enter valid URL </div>`);
```

Comment:
```suggestion
if (urlInput.parent().children().length === 2) {
				urlInput.parent().append(`<div class="invalid-feedback" id="inputUrl-feedback0">
					Enter valid URL </div>`);
			}
```
It is better to check, whether there already exists an error message, and append only if needed. Currently, if the user clicks on 'Upload' button with invalid URL 6-7 times, the error message gets append 6-7 time , as shown in this **screenshot** -
![Screenshot from 2020-04-10 11-14-21](https://user-images.githubusercontent.com/45796806/78967561-73314480-7b20-11ea-8aaa-e8e905b0b092.png)